config/prod/iatlas-db.yaml

```diff
@@ -0,0 +1,18 @@
+template_path: aurora-postgresql-db.yaml
+stack_name: iatlasdb-prod
+dependencies:
+  - common/iatlasvpc.yaml
+  - prod/iatlas-kms-prod.yaml
+stack_tags:
+  Department: "CompOnc"
+  Project: "iAtlas"
+  OwnerEmail: "andrew.lamb@sagebase.org"
+parameters:
+  VpcName: iatlasvpc
+  KmsKeyName: "/alias/iatlas-kms-prod/KmsKey"
```

Comment:
This is just the key name correct?  is this really a secret?

Reply:
Hmm, the key alias is not a secret, just a way to reference the KMS Key... am I missing something?

Reply:
ohh ok.  for some reason i thought i was using the !ssm resolver.  looks good